src/test/cypress/e2e/exam/ExamAssessment.cy.ts

```diff
@@ -95,101 +87,125 @@ describe('Exam assessment', () => {
 
         it('Assess a programming exercise submission (MANUAL)', () => {
             cy.login(tutor, '/course-management/' + course.id + '/exams');
-            examManagement.openAssessmentDashboard(exam.id!, examDuration);
+            examManagement.openAssessmentDashboard(exam.id!, 155000);
             startAssessing();
             examAssessment.addNewFeedback(2, 'Good job');
             examAssessment.submit();
             cy.login(studentOne, '/courses/' + course.id + '/exams/' + exam.id);
             programmingExerciseEditor.getResultScore().should('contain.text', '66.2%, 6 of 13 passed, 6.6 points').and('be.visible');
+            programmingAssessmentSuccessful = true;
+        });
+
+        it('Complaints about programming exercises assessment', () => {
+            if (programmingAssessmentSuccessful) {
+                handleComplaint(false, CypressExerciseType.PROGRAMMING);
+            }
```

Comment:
Should we let this test fail, if it was not able to verify that handling complaints work?
Same for the other tests down below with an if-clause

Reply:
Cypress should fail if any of the steps contained in the `handleComplaint` function fails, so this should already work like that